src/common/CoreMediaIO/DeviceAbstractionLayer/Devices/Sample/PlugIn/CMIO_DP_Sample_Stream.cpp

```diff
@@ -119,7 +121,7 @@ namespace CMIO { namespace DP { namespace Sample
 		mScheduledOutputNotificationProc(NULL),
 		mDeck(NULL),
 		mFormatPairs(),
-		mFrameType(DPA::Sample::kYUV422_1280x720),
+		mFrameType(getFrameType()),
```

Comment:
I would just remove all the code in this file, and revert back to simply `mFrameType(DPA::Sample::kYUV422_1280x720)`

Then you could also leave a comment that this format doesn't have any effect and we're not sure why.

Reply:
yeah sure